packages/nimble-components/src/patterns/dropdown/styles.ts

```diff
@@ -142,12 +142,10 @@ export const styles = css`
         box-sizing: border-box;
         display: inline-flex;
         flex-direction: column;
-        left: 0;
         overflow-y: auto;
         width: 100%;
         --ni-private-listbox-padding: ${smallPadding};
         max-height: calc(var(--ni-private-select-max-height) - ${smallPadding});
-        z-index: 1;
```

Comment:
This has no effect because the `position` of the element is `static` (the default).